### ISSUE TITLE
[Feature request] Dropdown Color Selector

### ISSUE THREAD

**Issue by cuiliang** [HandyOrg/HandyControl#768]
1. Work like combobox
2. Preview color as background or small box.
3. Show color value text.
4. 取色器支持自定义代码实现。目前的取色器 ![image](https://user-images.githubusercontent.com/1972649/113831840-9ede1d80-97ba-11eb-863d-c51e300d2c68.png) 只支持在当前窗口里取色，不支持桌面任意位置。
5. 确定和取消按钮的位置，和MessageBox里是反的，感觉统一一下比较好。

![image](https://user-images.githubusercontent.com/1972649/113831639-676f7100-97ba-11eb-9311-87acf99691e7.png)

**Comment by ghost1372** [comment]
I do not think this is really necessary The user can achieve this by editing the Combobox template themselves In the case of ColorPicker, in the unofficial version, you can choose the color from anywhere in Desktop
![0](https://user-images.githubusercontent.com/9213496/113834229-d8555f80-979f-11eb-85d5-d4bbcbf50405.gif)

**Comment by cuiliang** [comment]
Color picker is commonly used.  If we have this control, it will be very Handly :)

The following image is a demo of customizing UI style. Using control from [wpftoolkit](https://github.com/xceedsoftware/wpftoolkit/wiki/ColorPicker). which sytle is not compatable with handycontrol😂

![image](https://user-images.githubusercontent.com/1972649/113857655-5b44dd00-97d5-11eb-855a-5a20830afba5.png)

**Comment by ghost1372** [comment]
I can see in the first picture that you have implemented this yourself So why not send a PR?

**Comment by cuiliang** [comment]
> I can see in the first picture that you have implemented this yourself So why not send a PR?

no....😂  just combined some small images, not real control.   It's beyond my capability now.
